src/templates/filtersDrawer.jsx

```diff
@@ -111,6 +129,13 @@ const propTypes = {
             toggle: PropTypes.shape({
                 checked: PropTypes.bool,
                 className: PropTypes.string,
+                /**
+                 * A DatePickerInput can be disabled.
```

Comment:
Same here; please reword for this placement versus copy-and-paste source.

Reply:
Fixing...

Reply:
Commit 7c8b691 fixes this.

Reply:
Looks good.  Resolving.